### PR TITLE
Adds getMatchingProductForId

### PR DESCRIPTION
**Pull request by MarkVaughn**
This is what promote requires to replace phorizon.
Will not work without #1

### CODE REVIEW

File: src/SellerLabs/Mws/Mws.php [hunk 1]
```diff
@@ -70,55 +91,110 @@ private function getProductsClient()
...
-                    'MarketplaceId' => $this->marketplaceId,
-                    'IdType' => 'ASIN',
-                    'IdValue' => $asin,
-                    'Identifier' => 'identifier',
-                    'IsAmazonFulfilled' => true,
-                    'PriceToEstimateFees' => [
-                        'ListingPrice' => [
-                            'CurrencyCode' => 'USD',
-                            'Amount' => $listingPrice,
+        $request = new GMF(
```
Comment (MarkVaughn) [inline src/SellerLabs/Mws/Mws.php]: 🌵 Formatting

File: src/SellerLabs/Mws/Mws.php [hunk 2]
```diff
@@ -70,55 +91,110 @@ private function getProductsClient()
...
+        );
 
-        $response = $this->getProductsClient()->GetMyFeesEstimate($request);
+        $response = $this->getProductsClient()->getMyFeesEstimate($request);
+
+        $xml = simplexml_load_string(
+            $response->toXML()
+        )->GetMyFeesEstimateResult->FeesEstimateResultList->FeesEstimateResult;
+
+        $this->throwExceptionOnError($xml);
```
Comment (MarkVaughn) [inline src/SellerLabs/Mws/Mws.php]: extracted to reuse

File: src/SellerLabs/Mws/Mws.php [hunk 3]
```diff
@@ -43,13 +48,28 @@ public function __construct(Repository $config)
...
 
     /**
+     * Set the marketplace Id
+     *
+     * @url http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html
+     * @param $marketplaceId
+     *
+     * @return $this
+     */
+    public function setMarketplaceId($marketplaceId)
```
Comment (MarkVaughn) [inline src/SellerLabs/Mws/Mws.php]: Promote uses all marketplaces

File: src/SellerLabs/Mws/Mws.php [hunk 4]
```diff
@@ -14,7 +14,10 @@
 use Illuminate\Contracts\Config\Repository;
 use MarketplaceWebServiceProducts_Client;
 use MarketplaceWebServiceProducts_Mock;
-use MarketplaceWebServiceProducts_Model_GetMyFeesEstimateRequest;
+use MarketplaceWebServiceProducts_Model_GetMyFeesEstimateRequest as GetMyFeesEstimateRequest;
```
Comment (dsh) [inline src/SellerLabs/Mws/Mws.php]: 👍

**Pull request merged**